src/Files.App/Actions/Open/OpenPropertiesAction.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using Files.App.Commands;
+using Files.App.Contexts;
+
+namespace Files.App.Actions
+{
+	internal class OpenPropertiesAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+		public string Label { get; } = "OpenProperties".GetLocalizedResource();
+
+		public string Description => "OpenPropertiesDescription".GetLocalizedResource();
+
+		public RichGlyph Glyph { get; } = new RichGlyph(opacityStyle: "ColorIconProperties");
+
+		public bool IsExecutable =>
+			context.ShellPage.InstanceViewModel.IsPageTypeNotHome &&
+			context.HasSelection;
```

Comment:
Shouldn't the command be usable on the Recent Files, which is on the Home Page?

Reply:
We should handle it very carefully, since it may cause bugs (see #12220)
Currently, if you have nothing selected you can click properties button to open parent folder ones. The condition above is blocking that behaviour

Reply:
Then always executable?

Reply:
I think we should check only if page is not home.
We should handle recent files differently, as we would do for sidebar items